### ISSUE TITLE
As a bioinformatician working with production analysis, I want the bug where cases missing information in status DB blocks the automatic start of the following analysis in the queue fixed to minimize manual intervention

### ISSUE THREAD

**Issue by emiliaol** [Clinical-Genomics/cg#105]
If a case that's marked for analysis (or that has no status at all) in the family table in status DB miss any information like capture kit cg analysis auto will throw a python error and exit. This aborts the whole process and it does not try to start any subsequent cases.

Actions:
* Add a warning that's sent to a suitable place so we can pick these errors up from the crontab (this will be fixed in a related user story)
* Make the process skip and not exit on the error so it moves on to the next  case

**DoD:**
See patch DoDs
Automated test not needed if manual test will suffice.

**Comment by barrystokman** [comment]
Unclear what the error message is, or how to reproduce the defect. Cannot be solved for now

**Comment by ingkebil** [comment]
Could you also add an `angrybird` so we can check if this now works?

**Comment by emiliaol** [comment]
Found one! alivebedbug
Please don't actually start the analysis of this (or make sure to cancel it if you do). I have not set this to analyze in status db so please do so when you want to.

**Comment by ingkebil** [comment]
I need to set it to analyze to test the automation. Just ot make sure: this one will fail, right? I need an analysis that fails on missing LIMS info

**Comment by ingkebil** [comment]
So, `alivebedbug` will not autostart because it contains samples that are externally sequenced ;)

That being said, output from `cg get family alivebedbug` seems to be wrong in that aspect.

**Comment by emiliaol** [comment]
I didn't check it that thoroughly. Just picked it from the log... Anyway. I created a new family for you to test called **prostud** that will fail on this:

```(prod170926) [hiseq.clinical@rastapopoulos ~]$ cg analysis auto
prostud: start analysis
Traceback (most recent call last):
  File "/mnt/hds/proj/bioinfo/SERVER/miniconda/envs/prod170926/bin/cg", line 11, in <module>
    load_entry_point('cg==1.4.6', 'console_scripts', 'cg')()
  File "/mnt/hds/proj/bioinfo/SERVER/miniconda/envs/prod170926/lib/python3.6/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/mnt/hds/proj/bioinfo/SERVER/miniconda/envs/prod170926/lib/python3.6/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/mnt/hds/proj/bioinfo/SERVER/miniconda/envs/prod170926/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/mnt/hds/proj/bioinfo/SERVER/miniconda/envs/prod170926/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/mnt/hds/proj/bioinfo/SERVER/miniconda/envs/prod170926/lib/python3.6/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/mnt/hds/proj/bioinfo/SERVER/miniconda/envs/prod170926/lib/python3.6/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/mnt/hds/proj/bioinfo/SERVER/miniconda/envs/prod170926/lib/python3.6/site-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/mnt/hds/proj/bioinfo/SERVER/miniconda/envs/prod170926/lib/python3.6/site-packages/cg/cli/analysis.py", line 153, in auto
    context.invoke(analysis, priority=priority, family_id=family_obj.internal_id)
  File "/mnt/hds/proj/bioinfo/SERVER/miniconda/envs/prod170926/lib/python3.6/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/mnt/hds/proj/bioinfo/SERVER/miniconda/envs/prod170926/lib/python3.6/site-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/mnt/hds/proj/bioinfo/SERVER/miniconda/envs/prod170926/lib/python3.6/site-packages/cg/cli/analysis.py", line 52, in analysis
    context.invoke(config, family_id=family_id)
  File "/mnt/hds/proj/bioinfo/SERVER/miniconda/envs/prod170926/lib/python3.6/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/mnt/hds/proj/bioinfo/SERVER/miniconda/envs/prod170926/lib/python3.6/site-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/mnt/hds/proj/bioinfo/SERVER/miniconda/envs/prod170926/lib/python3.6/site-packages/cg/cli/analysis.py", line 75, in config
    config_data = context.obj['api'].config(family_obj)
  File "/mnt/hds/proj/bioinfo/SERVER/miniconda/envs/prod170926/lib/python3.6/site-packages/cg/meta/analysis.py", line 88, in config
    data = self.build_config(family_obj)
  File "/mnt/hds/proj/bioinfo/SERVER/miniconda/envs/prod170926/lib/python3.6/site-packages/cg/meta/analysis.py", line 120, in build_config
    capture_kit = self.lims.capture_kit(link.sample.internal_id)
  File "/mnt/hds/proj/bioinfo/SERVER/miniconda/envs/prod170926/lib/python3.6/site-packages/cg/apps/lims/api.py", line 117, in capture_kit
    raise LimsDataError(message)
cg.exc.LimsDataError```

**Comment by ingkebil** [comment]
Fixed in 5234a8d9b6be44a3bed
